Glossary.md

```diff
@@ -295,6 +299,10 @@
 	Strategic Integration Platform acts as a central controller for all electronic messages sent within the 
     Trust. It routes messages from various sources ([POC](#poc), [HSL](#hsl), Imaging) to be recorded in the 
     appropriate databases.
+
+- <a id="star">**SPC**</a>
+
+	Statistical Process Chart (or Control).

```

Comment:
```suggestion
	Statistical Process Chart Control.

```
I think they are referred to as SPC charts so the C is control. This matches how they are described in out [InformUs rules documents](https://github.com/inform-us/requirements_specifications/blob/main/rules/SPCchartbuild.md)

Reply:
I now think you're right. I was just going on this...

![Screenshot 2024-01-25 at 15 21 24](https://github.com/inform-health-informatics/emap_documentation/assets/1836192/2268c6a2-108e-4840-9856-37070c06af73)